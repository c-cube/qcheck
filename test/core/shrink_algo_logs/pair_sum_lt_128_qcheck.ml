open QCheck

let pair_sum_lt_128 =
  Test.make ~name:"pairs sum to less than 128"
    (pair pos_int pos_int) (Log.shrinks Print.(pair int int) @@ fun (i,j) -> i+j<128)
;;
try Test.check_exn pair_sum_lt_128 with Test.Test_fail _ -> ()
