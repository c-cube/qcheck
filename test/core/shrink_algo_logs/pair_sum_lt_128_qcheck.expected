holds (1526421322787444331, 4437769643960039450)
holds (3367339764960236310, 4222330705308310169)
holds (2790778446709483554, 2587213195390812545)
holds (3066594532144543221, 2606114362798169773)
fails (617863026503364673, 924354758750856816)
fails (308931513251682337, 924354758750856816)
fails (154465756625841169, 924354758750856816)
fails (77232878312920585, 924354758750856816)
fails (38616439156460293, 924354758750856816)
fails (19308219578230147, 924354758750856816)
fails (9654109789115074, 924354758750856816)
fails (4827054894557537, 924354758750856816)
fails (2413527447278769, 924354758750856816)
fails (1206763723639385, 924354758750856816)
fails (603381861819693, 924354758750856816)
fails (301690930909847, 924354758750856816)
fails (150845465454924, 924354758750856816)
fails (75422732727462, 924354758750856816)
fails (37711366363731, 924354758750856816)
fails (18855683181866, 924354758750856816)
fails (9427841590933, 924354758750856816)
fails (4713920795467, 924354758750856816)
fails (2356960397734, 924354758750856816)
fails (1178480198867, 924354758750856816)
fails (589240099434, 924354758750856816)
fails (294620049717, 924354758750856816)
fails (147310024859, 924354758750856816)
fails (73655012430, 924354758750856816)
fails (36827506215, 924354758750856816)
fails (18413753108, 924354758750856816)
fails (9206876554, 924354758750856816)
fails (4603438277, 924354758750856816)
fails (2301719139, 924354758750856816)
fails (1150859570, 924354758750856816)
fails (575429785, 924354758750856816)
fails (287714893, 924354758750856816)
fails (143857447, 924354758750856816)
fails (71928724, 924354758750856816)
fails (35964362, 924354758750856816)
fails (17982181, 924354758750856816)
fails (8991091, 924354758750856816)
fails (4495546, 924354758750856816)
fails (2247773, 924354758750856816)
fails (1123887, 924354758750856816)
fails (561944, 924354758750856816)
fails (280972, 924354758750856816)
fails (140486, 924354758750856816)
fails (70243, 924354758750856816)
fails (35122, 924354758750856816)
fails (17561, 924354758750856816)
fails (8781, 924354758750856816)
fails (4391, 924354758750856816)
fails (2196, 924354758750856816)
fails (1098, 924354758750856816)
fails (549, 924354758750856816)
fails (275, 924354758750856816)
fails (138, 924354758750856816)
fails (69, 924354758750856816)
fails (35, 924354758750856816)
fails (18, 924354758750856816)
fails (9, 924354758750856816)
fails (5, 924354758750856816)
fails (3, 924354758750856816)
fails (2, 924354758750856816)
fails (1, 924354758750856816)
fails (0, 924354758750856816)
fails (0, 462177379375428408)
fails (0, 231088689687714204)
fails (0, 115544344843857102)
fails (0, 57772172421928551)
fails (0, 28886086210964276)
fails (0, 14443043105482138)
fails (0, 7221521552741069)
fails (0, 3610760776370535)
fails (0, 1805380388185268)
fails (0, 902690194092634)
fails (0, 451345097046317)
fails (0, 225672548523159)
fails (0, 112836274261580)
fails (0, 56418137130790)
fails (0, 28209068565395)
fails (0, 14104534282698)
fails (0, 7052267141349)
fails (0, 3526133570675)
fails (0, 1763066785338)
fails (0, 881533392669)
fails (0, 440766696335)
fails (0, 220383348168)
fails (0, 110191674084)
fails (0, 55095837042)
fails (0, 27547918521)
fails (0, 13773959261)
fails (0, 6886979631)
fails (0, 3443489816)
fails (0, 1721744908)
fails (0, 860872454)
fails (0, 430436227)
fails (0, 215218114)
fails (0, 107609057)
fails (0, 53804529)
fails (0, 26902265)
fails (0, 13451133)
fails (0, 6725567)
fails (0, 3362784)
fails (0, 1681392)
fails (0, 840696)
fails (0, 420348)
fails (0, 210174)
fails (0, 105087)
fails (0, 52544)
fails (0, 26272)
fails (0, 13136)
fails (0, 6568)
fails (0, 3284)
fails (0, 1642)
fails (0, 821)
fails (0, 411)
fails (0, 206)
holds (0, 103)
fails (0, 155)
holds (0, 78)
holds (0, 117)
fails (0, 136)
holds (0, 68)
holds (0, 102)
holds (0, 119)
fails (0, 128)
holds (0, 64)
holds (0, 96)
holds (0, 112)
holds (0, 120)
holds (0, 124)
holds (0, 126)
holds (0, 127)
